flight/targets/brainre1/fw/Makefile

```diff
@@ -43,7 +43,7 @@ MODULES += Telemetry
 
 OPTMODULES += GPS
 OPTMODULES += PathPlanner
-OPTMODULES += AltitudeHold
+#OPTMODULES += AltitudeHold
```

Comment:
I think we prefer to delete this rather than comment it (so that grep, etc is more effective going forward)